src/main/java/uk/gov/hmcts/reform/sscs/config/HmcHearingsEventTopicListener.java

```diff
@@ -22,23 +27,59 @@
 @ConditionalOnProperty("flags.hmc-to-hearings-api.enabled")
 public class HmcHearingsEventTopicListener {
 
+
+    @Autowired
+    private static HearingsJourneyService hearingsJourneyService;
```

Comment:
Why is this static?

Reply:
It's come from master branch which edit by @zikrurR, I also have the same question in my mind.

Reply:
I don't think this is right - can we change it or raise a tech debt ticket?

Reply:
This should not be static